burnman/seismic.py

```diff
@@ -222,45 +280,59 @@ def v_s(self, depth):
 
         return self._lookup(depth, self.table_vs)
 
+    def QK(self, depth):
+
+        return self._lookup(depth, self.table_Qk)
+
+    def QG(self, depth):
+
+        return self._lookup(depth, self.table_Qmu)
+
 
     def density(self, depth):
 
         return self._lookup(depth, self.table_density)
 
     def depth(self, pressure):
-        if pressure > max(self.table_pressure) or pressure < min(self.table_pressure)  :
-           raise ValueError, "Pressure outside range of PREM"
-        radius = np.interp(pressure, self.table_pressure[::-1], self.table_radius[::-1] )
-        return self.earth_radius - radius
+        if self.table_depth[0]:
```

Comment:
Should we no longer have this error check?  According to the documentation for interp, it returns the value on the edge of the model points if you request a point outside of the interval.